### DISCUSSION TITLE
Difference between Github Copilot and Github Copilot Nightly.

### DISCUSSION THREAD

**Discussion by Victorsitou** [community/community#7074] [Copilot Conversations] [answered]
Hi, while I was searching for Copilot on Visual Studio Code I noticed that there are two versions of it, the normal one, and a "Nightly" version. My question is, what's the difference between these two versions?

**Comment by davecheney** [comment, marked as answer]
The nightly build contains newer, more experimental, perhaps less well tested, changes.

**Reply by stogaja** [reply]
So Which one is better, copilot or copilot nightly?

**Reply by davecheney** [reply]
They’re the same, but the nightly has changes that are committed sooner.

**Comment by LastGentlman** [comment]
haa.. So Nightly is like a beta version

**Reply by patrick-amusivwa** [reply]
yes

**Reply by 8xu** [reply]
Exactly!

**Comment by chenrui333** [comment]
One thing to note is the [Github Copilot Labs](https://githubnext.com/projects/copilot-labs/) was bundled with Github Copilot nightly before.

> Are you a [Copilot Nightly](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly) user? You might have used a previous version of Labs which was bundled within that extension. We've ported all of those features from Copilot Nightly to this separate extension.

**Reply by olawalejuwonm** [reply]
Does Copilot Nightly have same features as Github Copilot Labs?

**Reply by davecheney** [reply]
no

**Comment by olawalejuwonm** [comment]
So what's the difference?

**Reply by davecheney** [reply]
It’s a beta release.
